src/python/WMCore/MicroService/Unified/Transferor.py

```diff
@@ -168,37 +384,56 @@ def add(self, requests):
                 datasets = rdict.get('datasets')
                 sites = rdict.get('sites')
                 subscription = PhEDExSubscription(datasets, sites, self.group)
-                if self.verbose:
-                    print("### add subscription", subscription)
+                self.logger.debug("### add subscription %s" % subscription)
                 # TODO: when ready enable submit subscription step
                 # self.phedex.subscribe(subscription)
 
+		# TODO: when we made subscription we change status of request in ReqMgr2
+
     def info(self, request=None):
         "Return info about given request"
         if not request:
             return self.store.info()
-        completed = self.checkPhedex(request)
+        completed = self.checkSubscription(request)
         idict = self.store.info(request)
-        idict.update({'completed': completed})
+        idict.update({'status': completed})
         return idict
 
     def delete(self, request):
         "Delete request in backend"
         return self.store.delete(request)
 
+class Services(object):
+    "Services class provides access to reqmgr2 services: ReqMgr, ReqMgrAux"
+    __slots__ = ('reqmgrAux', 'reqmgr', 'workqueue')
+    def __init__(self, reqmgrUrl, workqueueUrl, logger=None):
+        self.reqmgrAux = ReqMgrAux(reqmgrUrl, logger=logger)
+        self.reqmgr = ReqMgr(reqmgrUrl, logger=logger)
+
 class UnifiedTransferorManager(object):
     """
     Initialize UnifiedTransferorManager class
     """
-    def __init__(self, config=None):
+    def __init__(self, config=None, logger=None):
         self.config = config
-        self.reqmgrAux = ReqMgrAux(self.config.reqmgr2_url)
+        if logger:
+            self.logger = logger
+        else:
+            self.logger = logging.getLogger('reqmgr2ms:UnifiedTransferorManager')
```

Comment:
How about replacing it by
`logging.getLogger(self.__class__.__name__)`
?

Reply:
ok